llama_stack/providers/tests/eval/test_eval.py

```diff
@@ -131,7 +131,7 @@ async def test_eval_run_eval(self, eval_stack, inference_model, judge_model):
 
     @pytest.mark.asyncio
     async def test_eval_run_benchmark_eval(self, eval_stack, inference_model):
-        eval_impl, benchmarks_impl, datasets_impl, models_impl = (
```

Comment:
line 138 could be removed then? (same above)

Reply:
Addressed the entire file, thanks!